project/Settings.scala

```diff
@@ -62,7 +62,6 @@ object Settings {
     val flywayVersion           = "6.0.4"
 
     // test libraries
-    val scalaTest               = "3.0.5"
     val scalaCheck              = "1.13.5"
```

Comment:
I guess we don't need scalaCheck nor discipline anymore

Reply:
Ok, removed them too.